credsweeper/rules/config.yaml

```diff
@@ -246,13 +246,13 @@
   severity: medium
   type: keyword
   values:
-    - pass|pwd
+    - pass|pw(d|\b)
```

Comment:
Using `\b` for checking word boundary is good :eyes: 
I hope this change doesn't make much FPs in real world..